src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public double getArea();
```

Comment:
```suggestion
    double getArea();
```
У інтерфейсі усі методи по дефолту public abstract